Cue/app/setup.js

```diff
@@ -29,6 +30,7 @@ function setup(): ReactClass<{}> {
       if (this.state.isLoading) {
         return null;
       }
+      CueApi.setAuthHeader(this.state.store.getState().user.userId, this.state.store.getState().user.accessToken);
```

Comment:
I tried that at first, but this.store is undefined during the callback. Not sure why, the debugger says they are defined and I can access them, but react-native throws a `Cannot read property 'getState' of undefined`